release/pkg/utils/utils.go

```diff
@@ -27,6 +27,7 @@ import (
 const hexAlphabet = "0123456789abcdef"
 
 func ExecCommand(cmd *exec.Cmd) (string, error) {
+	fmt.Println(cmd.String())
```

Comment:
Should this be written via a log statement or behind a debug flag or something? Or did you mean to leave this in just like this?

Reply:
Oh my bad this was used for testing. I'll remove it.